### PR TITLE
ENH - map filter and tiles as an activity from Map Fragment

### PR DESCRIPTION
**Pull request by rksh**
first cut at changing map settings to own dialog, combining with maptile settings.

template for clarifying main settings mage, modularizing settings that appear in more than one place.

### CODE REVIEW

File: wiglewifiwardriving/src/main/java/android/support/v4/app/FixedMapSavedState.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import android.os.Bundle;
+import android.os.Parcel;
+import android.support.v4.app.Fragment;
+
+/**
+ * Created by arkasha on 7/3/17.
+ */
+
+public class FixedMapSavedState extends Fragment.SavedState {
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/android/support/v4/app/FixedMapSavedState.java]: oh my, what's the story behind this thing?

Comment (rksh) [inline wiglewifiwardriving/src/main/java/android/support/v4/app/FixedMapSavedState.java]: having just completed the gingerbread back-compat change, I tried this fix on my Gingerbread Nexus S instance - to find that I hadn't completely fixed a badParcelable exception with maps; this change was a provisional diff I had hanging around from efforts earlier in the month, fixes in emulation.

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/MappingFragment.java [hunk 2]
```diff
@@ -87,11 +87,14 @@
     private AtomicBoolean finishing;
     private Location previousLocation;
     private int previousRunNets;
+    private TileOverlay tileOverlay;
```
Comment (rksh) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/MappingFragment.java]: shoving this out to the class level so we can mess with it onResume

File: wiglewifiwardriving/src/main/res/layout/settings.xml [hunk 3]
```diff
@@ -256,20 +213,21 @@
...
 		/>
 	<TextView
 		android:id="@+id/gps_text"
-		android:layout_width="wrap_content"
-		android:layout_height="wrap_content"
+		android:layout_width="0dp"
+		android:layout_height="match_parent"
 		android:text="@string/gps_text"
-		android:layout_weight="1"
+		android:layout_weight="2"
```
Comment (rksh) [inline wiglewifiwardriving/src/main/res/layout/settings.xml]: so, in languages where "portuguese (brazilian) becomes too wide for the screen, we force the label text into a 2DP column at the far right of the screen. This prevents that.

**Pull request merged**